### PR TITLE
Fix ppca test

### PR DESCRIPTION
**Pull request by ordabayevy**
Use biased (normalization factor=`n`) covariance to perform expected PCA.

### CODE REVIEW

File: test/test_ppca.py [hunk 1]
```diff
@@ -55,7 +47,7 @@ def test_probabilistic_pca_multi_device(
     train_loader = torch.utils.data.DataLoader(
         TestDataset(x_ng),
         batch_size=batch_size,
-        shuffle=True,
+        shuffle=False,
```
Comment (ordabayevy) [inline test/test_ppca.py:50]: PyTorch's sampler behaves strangely with `devices=2`. Disabling shuffling resolves the issue and PPCA achieves high accuracy fit.

**Review by mbabadi — APPROVED** [review]
